src/microbe_stage/systems/StrainSystem.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Components;
+using DefaultEcs;
+using DefaultEcs.System;
+using DefaultEcs.Threading;
+using Godot;
+using World = DefaultEcs.World;
+
+[With(typeof(OrganelleContainer))]
+[With(typeof(StrainAffected))]
+[ReadsComponent(typeof(OrganelleContainer))]
+[ReadsComponent(typeof(MicrobeControl))]
+public sealed class StrainSystem : AEntitySetSystem<float>
+{
+    public StrainSystem(World world, IParallelRunner runner) : base(world, runner)
+    {
+    }
+
+    protected override void Update(float delta, in Entity entity)
+    {
+        ref var strain = ref entity.Get<StrainAffected>();
+        ref var organelles = ref entity.Get<OrganelleContainer>();
```

Comment:
This should only be fetched in the if condition where used to increase performance of this system.